### PR TITLE
prototype_view: sprint 1 commit

### PR DESCRIPTION
**Pull request by haydenkarl22**
Sprint 1 Progress: Hayden Karl

The issue I'm working on currently requires me to design some prototypes of the view of the user for this 
application. Our group met with Dr. McCulla, a professor in the Chemistry department to get his feel for the look
and some desired functionality of these prototypes. Saved in this dir is a screenshot of the current view of the 
ORCA file converter.

Dr. McCulla didn't suggest any styling for the page, however he did inform me that the current elements already on
the view, such as the file selector button, the section selection dropdown tab, and the file name text box are all
features he wants to stay the same, with maybe slight optimizations for the backend as needed. He also wanted the 
ability to enter in keywords that the application would search for and use the data that is under the keywords, so 
another textbox that takes keywords will be necessary. This, however, is all he suggested, so I'll design a couple
of simple pages with these suggestions in mind while also adding some stylings of my own. My stylings won't detract
from the functionality or use of the page. My goal is to make this page look more like something a company or school
would deploy for public use rather than a barebones page. 

The directory for the prototypes is made as stated in the root directory.

### CODE REVIEW

File: Prototypes/SprintOneReadMe.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Sprint 1 Progress: Hayden Karl
```
Comment (kate-holdener) [inline Prototypes/SprintOneReadMe.txt:1]: @haydenkarl22 We don't want to include 'sprint' specific files. I suggest pasting the contents of this ReadMe file into the pull request comment and removing this SprintOneReadMe.txt file. Also, all commits you make are tied to your GitHub profile, so you don't need to include your name on them (we know who made the commits and who created the PRs)

**Comment by kate-holdener** [comment]
> Sprint 1 Progress: Hayden Karl
> 
> The issue I'm working on currently requires me to design some prototypes of the view of the user for this application. Our group met with Dr. McCulla, a professor in the Chemistry department to get his feel for the look and some desired functionality of these prototypes. Saved in this dir is a screenshot of the current view of the ORCA file converter.
> 
> Dr. McCulla didn't suggest any styling for the page, however he did inform me that the current elements already on the view, such as the file selector button, the section selection dropdown tab, and the file name text box are all features he wants to stay the same, with maybe slight optimizations for the backend as needed. He also wanted the ability to enter in keywords that the application would search for and use the data that is under the keywords, so another textbox that takes keywords will be necessary. This, however, is all he suggested, so I'll design a couple of simple pages with these suggestions in mind while also adding some stylings of my own. My stylings won't detract from the functionality or use of the page. My goal is to make this page look more like something a company or school would deploy for public use rather than a barebones page.
> 
> The directory for the prototypes is made as stated in the root directory.

@haydenkarl22 You should add the features that the client wants to the prototype (ability to enter in keywords, example of data that matched the keywords). I wouldn't worry about styling for now.

**Comment by haydenkarl22** [comment]
Added one prototype of the home page under prototypes, along with screenshots of what's changed (those files are labeled as prototypes, the other screenshot is the current view). I sent my screenshots to Prof. McCulla for his approval, so I'm currently waiting for his input.

**Comment by rkarmuri** [comment]
I see the prototypes directory exists at the root of the repo. I would need instructions to run your code in the Read Me file of your branch. Also, copy the whole team when you are emailing the client to make sure we all are on the same page.

**Comment by haydenkarl22** [comment]
Hey, I took your advice and added a readme file that explains how to launch localhost with the prototype view, Let me know if there are any more comments or concerns. Also, since I haven't had a response from Mr. McCulla, I'll send another email to him and his student assistant, and I'll cc you in the email. Thanks for the heads up.

**Comment by rkarmuri** [comment]
@haydenkarl22 Based on your instructions, I removed the files in the views folder and added your current code to it, and it works fine. However, there is a small syntax error in your HTML div tag in the 1st line of your code. I will merge the PR as it looks good to me. Moving further, could you make the Read Me file more readable and instructive with commands to be issued and output expected? Also, you'll be able to change the current code in the front-end views folder to your latest code, which I will add as a separate issue. The reason is the command: ng serve picks from the front-end folder where we have the JSON file. Overall, great effort, and I appreciate your work.

**Review by rkarmuri — APPROVED** [review]
Adding the prototypes set up.

**Pull request merged**